### ISSUE TITLE
REACT: Mappings not showing where there is conflict

### ISSUE THREAD

**Issue by HayleyMills** [CLOSER-Cohorts/archivist#490]
Tested using alspac_08_DCSF which has question-variables mapped and topics. qc_1 is mapped to CCXD050 this had a topic Finances attached, I changed it to another topic and receieved the Error stating that the variable was already mapped- which is great, but it is mapped to Employment and pensions not Finances which is what it was originally mapped to? I would have expected there to already have been a conflict error. 

![image](https://user-images.githubusercontent.com/16664703/105489881-de14cc00-5cab-11eb-8edc-208f8b481f69.png)
![image](https://user-images.githubusercontent.com/16664703/105490140-3fd53600-5cac-11eb-9333-5007cf2079c8.png)

In addtion, it should show the topics which have been inherited- if you look at qc_2_i then it has no topic, but the variable it is mapped to is mapped to topic so it should inherit that topic. 
![image](https://user-images.githubusercontent.com/16664703/105489915-e967f780-5cab-11eb-8c38-c122d2d3ccc2.png)

**Comment by simonreed** [comment]
This is now deployed to heroku-react.

**Comment by HayleyMills** [comment]
Tested using alspac_08_DCSF again. The conflicts are showing when when the variables/topics are changes and they are already mapped. 

However, it should show qc_1 mapped to Employment and income (Employment and pensions)
![image](https://user-images.githubusercontent.com/16664703/111469137-40bf9c80-871e-11eb-8222-a7354c6c082b.png)

I added topic Gender to qc_2 so it should show CCXD100 mapped to gender
![image](https://user-images.githubusercontent.com/16664703/111469423-972cdb00-871e-11eb-8ab5-b3e097a36085.png)

I can't test this fully until I am able to load in the mapping files and able to change the topic on the datasets.
